lms/templates/main.html

```diff
@@ -140,16 +120,7 @@
     </div>
 
     % if not disable_footer:
-        <%block name="footer">
-          ## Can be overridden by child templates wanting to hide the footer.
-          % if theme_enabled() and not is_microsite():
-            <%include file="theme-footer.html" />
-          % elif settings.FEATURES.get('IS_EDX_DOMAIN', False) and not is_microsite():
-              <%include file="footer-edx-v3.html" />
-          % else:
-            <%include file="${microsite.get_template_path('footer.html')}" />
-          % endif
-        </%block>
+        <%include file="themable-footer.html" />
```

Comment:
> header.html is very small and replaceable. footer.html has more in it, including copyright and logos that we'd like people to leave in place. We can look at this asymmetry and try to come up with a better solution.

Gotcha. Hmm, in that case, I think keeping these at the same level of UI hierarchy is important, how does ``header-app`` + ``footer-app`` or ``header-global`` + ``footer-global`` sound? I'd prefer the former personally as 

1. these elements are shared across the whole of LMS app (not Studio or other edX products/experiences)
2. the UX team is doing some discovery on what more globally-focused pieces of UI would look like (which have some additional responsibilities than the nav and content contained in here). When the time comes to build those elements, having these suffixed with ``-global`` may cause confusion.